style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  font-family: "Raleway", "Helvetica", sans-serif;
+  color: black;
+  background-color: white;
+}
+
+.resume__text {
+  font-size: 40px;
+  text-decoration: none;
+  color: black;
+}
+
+h3 {
+  font-size: 30px;
+  text-decoration: none;
+  color: black;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+.header {
+  max-width: 1024px;
+  display: flex;
+  height: 400px;
+}
+.resume {
+  height: 100%;
+  background-color: rgb(194, 184, 184);
+  flex: 2;
+  border: 1px solid black;
+  margin: 0 auto;
+}
+.header__text {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+.photo {
+  height: 100%;
+  background-color: rgb(194, 184, 184);
+  flex: 2;
+  border: 1px solid black;
+}
+
+.photo__img {
+  border-radius: 30%;
+  display: block;
+  margin: 0 auto;
+  margin-top: 50px;
+}
+
+.info {
+  height: 100%;
+  background-color: rgb(194, 184, 184);
+  flex: 2;
+  border: 1px solid black;
+}
+
+.wrap {
+  height: 100%;
+  background-color: rgb(194, 184, 184);
+}
+
+.info__text {
+  font-size: 30px;
+  line-height: 50px;
+  text-transform: uppercase;
+  padding-left: 10px;
+}
+
+.scills {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+.scills__list {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+.scills__text {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+
+footer {
+    position: relative;
+    background: rgb(194, 184, 184);
+    max-width: 1024px;
+    display: block;
+    margin: 0 auto;
+}
+
+.footer__content {
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -webkit-justify-content: space-between;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    padding: 41px 7%;
+}
+

```

Comment:
there is no need to use here vendor prefix because you focus browser is the lates chrome